### DISCUSSION TITLE
compatible with orange Pi zero ?

### DISCUSSION THREAD

**Discussion by TB1405** [onlaj/Piano-LED-Visualizer#425] [Q&A]
Hi would this Programm also work with the Orange Pi Zero 512MB 
Best regards

**Comment by onlaj** [comment]
Hello.
For sure it won't fit into the same case.
It seems the pins layout is also slightly different, so you will probably have to modify some code.

**Reply by NikoTumi** [reply]
I'm willing to give it a shot on the OrangePi Zero 2W. Do you know which part of the code that needs altering? I'm well versed with bash coding, never worked with a GPIO dev board tho.

**Reply by NikoTumi** [reply]
What is the SPI setup needed for? Is it for the display or for also operating the LEDs?

**Comment by NikoTumi** [comment]
So one big issue I've been having trouble with is enabling the SPI. I've researched and tried using chatgpt but none of those had solutions. :(
